config/all-in-one/values.yaml

```diff
@@ -0,0 +1,100 @@
+# -----------------------------------------------------------------------------
+# Argo CD parameters
+# -----------------------------------------------------------------------------
+argocd:
+
+  # The type of the cluster that Argo CD runs on, values includes:
+  # - kubernetes
+  # - openshift
+  cluster: openshift
+
+  # The Argo CD namespace that can be auto-resolved based on cluster type,
+  # or you can uncomment below parameter to specificy it explicitly.
+  # namespace: argocd
+
+  # Allow apps to be deployed on the same cluster where Argo CD runs.
+  allowLocalDeploy: true
+
+# -----------------------------------------------------------------------------
+# Cluster parameters
+# -----------------------------------------------------------------------------
+cluster:
+
+  # Specify whether or not to provision a cluster before install CP4WAIOps.
+  enabled: false
+
+  # Provider specific configuration
+  provider:
+    # Fyre configuration
+    fyre:
```

Comment:
We may need to think more about this: How to enable the checking for different providers with some condition checking.

Reply:
Sure, this can be supported when we have more providers to come in.